client/ayon_cinema4d/plugins/create/create_workfile.py

```diff
@@ -0,0 +1,147 @@
+import ayon_api
+from ayon_core.pipeline import CreatedInstance, AutoCreator, AYON_INSTANCE_ID
+from ayon_cinema4d.api.plugin import cache_instance_data
+from ayon_cinema4d.api import lib, plugin
+
+
+class CreateWorkfile(AutoCreator):
+    """Workfile auto-creator.
+
+    The workfile instance stores its data on the `AYON_CONTAINERS` collection
+    as custom attributes, because unlike other instances it doesn't have an
+    instance node of its own.
+
+    """
+    identifier = "io.ayon.creators.blender.workfile"
```

Comment:
This is strange identifier?

Reply:
You sure - are 100% correct :D thanks for spotting the obvious.

Reply:
Fixed with https://github.com/ynput/ayon-cinema4d/pull/1/commits/71c47f8abdaa93ec54b3ab98848a879cd86e132f